cmd/tsbs_generate_queries/databases/questdb/devops.go

```diff
@@ -106,11 +107,12 @@ func (d *Devops) GroupByTimeAndPrimaryTag(qi query.Query, numMetrics int) {
 func (d *Devops) GroupByOrderByLimit(qi query.Query) {
 	interval := d.Interval.MustRandWindow(time.Hour)
 	sql := fmt.Sprintf(`
-		SELECT timestamp AS minute,
```

Comment:
I'd rather keep this one as SAMPLE BY + LIMIT -5. That's to avoid a full scan. We could add a `SELECT * FROM (...) ORDER BY minute DESC` outer query to align the result set order with other databases.

Reply:
Makes sense. Let me resolve this one.